include/NovelRT.h

```diff
@@ -150,7 +150,9 @@ namespace NovelRT::Windowing {
 
 #include "NovelRT/Animation/AnimatorPlayState.h"
 #include "NovelRT/Animation/SpriteAnimatorFrame.h"
-#include "NovelRT/Maths/GeoVector.h"
+#include "NovelRT/Maths/GeoVector2.h"
+#include "NovelRT/Maths/GeoVector3.h"
+#include "NovelRT/Maths/GeoVector4.h"
 #include "NovelRT/Maths/GeoMatrix4.h"
```

Comment:
nit: should probably be `GeoMatrix4x4` in order to differentiate from `Matrix4x3` or other types that sometimes crop up.